src/Microsoft.Extensions.Logging.Abstractions/project.json

```diff
@@ -28,6 +28,22 @@
         "System.Runtime.Extensions": "4.1.0-*",
         "System.Runtime.InteropServices": "4.1.0-*"
       }
+    },
+    "netcore50": {
+      "dependencies": {
+        "Microsoft.NETCore.Platforms": {
```

Comment:
Yup. Until we switch off `--infer-runtimes`, restore fails unless this Ms.NetCore.Platforms is listed as a build dependency.